data/config/fidesops.toml

```diff
@@ -26,6 +26,8 @@ DRP_JWT_SECRET = "testdrpsecret"
 LOG_LEVEL = "DEBUG"
 
 [execution]
+CELERY_BROKER_URL = "redis://:testpassword@redis:6379"
+CELERY_RESULT_BACKEND = "redis://:testpassword@redis:6379"
```

Comment:
I missed this, is this other toml file just used for tests? Can we add a code comment about what this file is for?

Reply:
I was also surprised to find this. Yes this file is just used in one test to check we can load the config in from a different path. The catch is that because of the way we validate configs, we need all the required vars in here (lest it throws a `ValidationError` as it should). Hence it looking the same as our actual config file.

Will add a comment 👍